nilearn/plotting/displays/_projectors.py

```diff
@@ -203,9 +200,8 @@ def add_graph(
             if not (adjacency_matrix.mask == adjacency_matrix.mask.T).all():
                 symmetric = False
                 warnings.warn(
-                    "'adjacency_matrix' was masked with "
-                    "a non symmetric mask. A directed "
-                    "graph will be plotted."
+                    "'adjacency_matrix' was masked with a non symmetric mask. "
+                    "A directed graph will be plotted."
```

Comment:
```suggestion
                    "A directed graph will be plotted.", stacklevel=3
```



Changing from the default of stacklevel from 1, so that the function used by the user is shown in the warning.

See this for more info
https://youtu.be/tZSEZ2WG5w8?feature=shared&t=218